frontend/src/app/features/new-dashboard/benchmarks-tab/benchmarks-tab.component.html

```diff
@@ -1,9 +1,9 @@
 <app-new-dashboard-header
-  *ngIf="showBanner"
+  *ngIf="!isParentViewingSubsidiary"
   [tab]="'benchmarks'"
   [updatedDate]="tilesData?.meta.lastUpdated"
   data-html2canvas-ignore
-  data-testid="showBannerTestId"
+  data-testid="isParentViewingSubsidiaryTestId"
```

Comment:
DashboardHeaderTestId
```suggestion
  data-testid="dashboardHeader"
```